src/beacon-api/verify_pagination.py

```
@@ -0,0 +1,68 @@
+import requests
+import sys
+
+def verify_pagination():
+    url = "http://localhost:8000/airqlouds"
+    try:
+        response = requests.get(url)
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)